eth/filters/api_test.go

```diff
@@ -29,8 +29,8 @@ func TestUnmarshalJSONNewFilterArgs(t *testing.T) {
 	var (
 		fromBlock rpc.BlockNumber = 0x123435
 		toBlock   rpc.BlockNumber = 0xabcdef
-		address0                  = common.StringToAddress("70c87d191324e6712a591f304b4eedef6ad9bb9d")
-		address1                  = common.StringToAddress("9b2055d370f73ec7d8a03e965129118dc8f5bf83")
+		address0                  = common.BytesToAddress([]byte("70c87d191324e6712a591f304b4eedef6ad9bb9d"))
+		address1                  = common.BytesToAddress([]byte("9b2055d370f73ec7d8a03e965129118dc8f5bf83"))
```

Comment:
Shouldn't these be `HexToAddress`?

Reply:
I know the current code retains behavior, I'm arguing the old behavior was faulty too.